src/github.md

```diff
@@ -0,0 +1,37 @@
+# GitHub
+Github is where the Rust project hosts all of its code, as well as a large parts
+of its discussions.
+
+## Organisations
+
+- [`rust-lang`] — The Rust project organisation.
+- [`rust-embedded`] — The Embedded Working Group organisation.
+- [`rustwasm`] — The WebAssembly Working Group organisation.
+- [`rust-cli`] — The Command Line Application Working Group organisation.
+
+[`rust-lang`]: https://github.com/rust-lang
+[`rust-embedded`]: https://github.com/rust-embedded
+[`rustwasm`]: https://github.com/rustwasm
+[`rust-cli`]: https://github.com/rust-cli
+
+## Team Repositories
+
+- [Compiler Team](https://github.com/rust-lang/compiler-team)
+- [Core Team](https://github.com/rust-lang/core-team)
+- [Crates.io Team](https://github.com/rust-lang/crates-io-cargo-teams)
+- [Infrastructure Team](https://github.com/rust-lang/infra-team)
+- [Language Team](https://github.com/rust-lang/lang-team)
+- [Moderation Team](https://github.com/rust-lang/moderation-team)
+- [Release Team](https://github.com/rust-lang/release-team)
+
+
+## Administration FAQ
+
+> How do I create a new repository under the `rust-lang` organisation, or make
+  changes that require `admin` level permissions?
+
+You can contact a GitHub admin directly, or post the request in [`project leads
```

Comment:
Let's (for now) note that GitHub admins are core team members. This is completely true today, and is unlikely I think to become not true (i.e., it may eventually be a subset, but even that I'm not sure about).

Reply:
Added another question that answers that.